scala-diff/shared/src/main/scala/munit/diff/NextPrintable.scala

```diff
@@ -0,0 +1,8 @@
+package munit.diff
+
+/**
+ * Don't override this ever
```

Comment:
Would be great to mention why a bit

Reply:
No longer relevant, shouldn't be pushed in the first place :wink: